Removed confusing test variable naming

Marcosrico opened the pull request:
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

N/A

### Description

- [X] Here are some details about my PR, including screenshots of any UI changes:

Removed unnecessary and confusing variables. Sharding key is a concept for federatedzkclient which isn't being used in Cunina. Removing as it doesn't provide any value and can be confusing.

### Tests

- [x] The following tests are written for this issue:

(List the names of added unit/integration tests)

- The following is the result of the "mvn test" command on the appropriate module:

(If CI test fails due to known issue, please specify the issue and test PR locally. Then copy & paste the result of "mvn test" to here.)

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

xyuanlu commented on meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java:
```diff
@@ -57,9 +57,8 @@ public class TestZkMetaClient {
-  protected static final String TRANSACTION_TEST_KEY_PREFIX = "/sharding-key-0";
-  protected static String PARENT_PATH = TRANSACTION_TEST_KEY_PREFIX + "/RealmAwareZkClient";
-  protected static final String TEST_INVALID_PATH = TRANSACTION_TEST_KEY_PREFIX + "_invalid" + "/a/b/c";
+  protected static String PARENT_PATH = "/ZkClient";
```
I would suggest using more specific name. 
If the entry will not be used by transactional tests, we could have that in the var name as well as value.

Marcosrico replied:
Sounds good, will update!

desaikomal commented on meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java:
```diff
@@ -57,9 +57,8 @@ public class TestZkMetaClient {
-  protected static String PARENT_PATH = TRANSACTION_TEST_KEY_PREFIX + "/RealmAwareZkClient";
-  protected static final String TEST_INVALID_PATH = TRANSACTION_TEST_KEY_PREFIX + "_invalid" + "/a/b/c";
+  protected static String PARENT_PATH = "/transactionOpTestPath";
+  protected static final String TEST_INVALID_PATH = "_invalid" + "/a/b/c";
```
nit: just curious,  why not "_invalid/a/b/c" ?

Marcosrico replied:
Good point! Will fix

desaikomal reviewed (approved):
LGTM. thanks

qqu0127 commented on meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java:
```diff
@@ -57,9 +57,8 @@ public class TestZkMetaClient {
-  protected static final String TRANSACTION_TEST_KEY_PREFIX = "/sharding-key-0";
-  protected static String PARENT_PATH = TRANSACTION_TEST_KEY_PREFIX + "/RealmAwareZkClient";
-  protected static final String TEST_INVALID_PATH = TRANSACTION_TEST_KEY_PREFIX + "_invalid" + "/a/b/c";
+  protected static String PARENT_PATH = "/transactionOpTestPath";
+  protected static final String TEST_INVALID_PATH = "_invalid/a/b/c";
```
Not an issue, just asking, why protected? Do you plan to use these two elsewhere in the future?

Marcosrico replied:
Hmm good point, I guess not. Must've stuck since I translated a lot of the code from zkMulti test cases. I'll switch it to private - thanks!

qqu0127 reviewed (approved):
Overall LGTM

Marcosrico commented:
This PR is ready to merge, approved by @desaikomal and @qqu0127 
Commit message:
Modified transactionOp test case variable for clearer understanding. Added more comments to test cases.

xyuanlu commented on meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java:
```diff
@@ -57,9 +57,8 @@ public class TestZkMetaClient {
-  protected static final String TRANSACTION_TEST_KEY_PREFIX = "/sharding-key-0";
-  protected static String PARENT_PATH = TRANSACTION_TEST_KEY_PREFIX + "/RealmAwareZkClient";
-  protected static final String TEST_INVALID_PATH = TRANSACTION_TEST_KEY_PREFIX + "_invalid" + "/a/b/c";
+  private static String PARENT_PATH = "/transactionOpTestPath";
```
If they will only be used in transactional test, maybe `TRANSACTION_TEST_PARENT_PATH` ?

Marcosrico replied:
Sure, will fix!

xyuanlu reviewed (approved):
Thanks for the change. one nit.

Marcosrico commented:
This PR is ready to merge, approved by @desaikomal, @qqu0127, and @xyuanlu 
Commit message:
Modified transactionOp test case variable for clearer understanding. Added more comments to test cases.
